.gitmodules

```diff
@@ -4,3 +4,21 @@
 [submodule "lib/openzeppelin-contracts"]
 	path = lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
+[submodule "lib/dss-interfaces"]
```

Comment:
Not sure what is the best practice here, but we can optimize this like: https://github.com/ClioFinance/socgen-ofh-onboarding-prototype/pull/3/files#diff-6ea402a2ca81b3b6f5dba5eed6bfac5e5d8bf8660be870732ad6e202e1038a68R1